code/modules/clothing/under/trek.dm

```diff
@@ -0,0 +1,72 @@
+//Just some alt-uniforms themed around Star Trek - Pls don't sue, Mr Roddenberry ;_;
+
+
+/obj/item/clothing/under/trek
+	can_adjust = 0
+
+
+//TOS
+/obj/item/clothing/under/trek/command
+	name = "command uniform"
```

Comment:
I'm disappointed that you didn't call this an "enterprising uniform". Shame on you. ___SHAME.___